system/LINK/bin/fix_auth

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# description: ManageIQ fix_auth launcher
+
+cd /var/www/miq/vmdb
+bundle exec ruby tools/fix_auth.rb "$@"
```

Comment:
Ok, @kbrock confirmed that the changing of the directory is happening in a subshell so the directory change doesn't change the user's current working directory unless they try to `source path_to_fix_auth` which is not what we want them to do anyway.